Makefile

```diff
@@ -215,12 +215,14 @@ generate: ##@other Regenerate assets and other auto-generated stuff
 	go generate ./static ./static/migrations
 	$(shell cd ./services/shhext/chat && exec protoc --go_out=. ./*.proto)
 
-prepare-release: clean-release
+$(RELEASE_DIRECTORY)/status-go-desktop.zip:
```

Comment:
I am not fan of this pattern because if the file already exists, even if it's outdated, nothing will be done. Can we call this target `prepare-release-desktop`?